.gitlab-ci.yml

```diff
@@ -1,9 +1,40 @@
 stages:
   - unit-test
+  - build
 
 run-unit-tests:
   stage: unit-test
   # A copy of golang image in dockerhub.
-  image: $CNS_IMAGE_GOLANG_1_19
+  image: $CNS_IMAGE_GOLANG
   script:
   - make test
+
+build-images:
+  stage: build
+  # A copy of docker image in dockerhub.
+  image: $CNS_IMAGE_DOCKER
+  services:
+    # A copy of docker-dind image in dockerhub.
+    - $CNS_IMAGE_DOCKER_DIND
+  variables:
+    DOCKER_HOST: tcp://localhost:2376
+    DOCKER_TLS_CERTDIR: "/certs"
+    DOCKER_TLS_VERIFY: 1
+    DOCKER_CERT_PATH: "$DOCKER_TLS_CERTDIR/client"
+  before_script:
+    # Sleep is needed to make sure that the docker engine is up.
+    - sleep 60
```

Comment:
Can we have a better way to wait until the docker engine is up? We can improve this in the next PR.